### PR TITLE
Replace rule label by workflow

### PR DESCRIPTION
**Pull request by AsmaaELM**
Issue: https://scality.atlassian.net/browse/ZKUI-132

### CODE REVIEW

**Comment by AsmaaELM** [comment]
ping

**Review by JBWatenbergScality — APPROVED** [review]

**Comment by AsmaaELM** [comment]
ping

File: src/react/workflow/CreateWorkflow.tsx [hunk 1]
```diff
@@ -231,7 +231,7 @@ const CreateWorkflow = () => {
             <T.Group>
               <T.GroupContent>
                 <T.Row>
-                  <T.Key principal={true}> Rule Type </T.Key>
+                  <T.Key principal={true}> Workflow Type </T.Key>
```
Comment (Cuervino) [inline src/react/workflow/CreateWorkflow.tsx:234]: Just "Type".

File: src/react/workflow/ConfigurationTab.tsx [hunk 2]
```diff
@@ -405,7 +405,7 @@ function EditForm({
             <T.Group>
               <T.GroupContent>
                 <T.Row>
-                  <T.Key principal={true}> Rule Type </T.Key>
+                  <T.Key principal={true}> Workflow Type </T.Key>
```
Comment (Cuervino) [inline src/react/workflow/ConfigurationTab.tsx:408]: Just "Type".

**Pull request merged**